### ISSUE TITLE
Fix cancer type covariate when single cancer is held out

### ISSUE THREAD

**Issue by jjc2718** [greenelab/pancancer-evaluation#5]
See code in `pancancer_utilities/tcga_utilities.py`, specifically the `align_matrices` function. This won't work if the test set has a different cancer type composition than the training set, so there needs to be a way to preserve this across datasets.
